src/ApiKitClient.ts

```diff
@@ -2,21 +2,31 @@ import axios, { AxiosInstance, AxiosResponse, AxiosError } from 'axios';
 import { PaginatedResponse } from './types';
 
 type UnauthorizationCallback = () => void;
+type AuthTokenCallback = () => Promise<string>;
 
 export class ApiKitClient {
-  private static instance: AxiosInstance | null = null;
+  private static instance: AxiosInstance;
   private static unauthorizationCallback?: UnauthorizationCallback;
+  private static authTokenCallback?: AuthTokenCallback;
 
```

Comment:
authTokenCallback optional?

Reply:
Missed this by accident!
Made both callbacks not optional...